hawkbit-ui/src/main/resources/messages.properties

```diff
@@ -215,12 +215,18 @@ label.tag.name = Tag name
 label.configuration.auth.header = Allow targets to authenticate via a certificate authenticated by an reverse proxy
 label.configuration.auth.gatewaytoken = Allow a gateway to authenticate and manage multiple targets through a gateway security token
 label.configuration.auth.targettoken = Allow targets to authenticate directly with their target security token
-label.configuration.repository.autoclose.action = Autoclose running actions with new Distribution set assignment
+label.configuration.repository.autoclose.action = Autoclose running actions when a new distribution set is assigned
+label.configuration.repository.autocleanup.action = Automatically delete finished actions
```

Comment:
Would it make sense to phrase it "... _delete terminated actions_", since `finished` could be mixed up with the status? And we only delete `canceled` and `failed` right?

Reply:
Done